### PR TITLE
Eval system - Export and Cloud Operation FSF-4061 FSF-4250

### PR DESCRIPTION
**Pull request by BrunoDias201907828**
At the moment, you are unable to run the send_cloud.py since you need an authetication file, still the evaluator should run  and export the metrics to csv files

### CODE REVIEW

File: src/cloud_storage/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+This folder contains scripts for interacting with Google Cloud Storage buckets, specifically for uploading CSV files. The main scripts are:
+
+- `bucket_operations.py`: Contains functions to upload and download files to/from a Google Cloud Storage bucket.
+- `send_bucket.py`: Uploads CSV files from unit tests and evaluations to the bucket.
+
+## Usage
+
+First run the following command
+```sh
```
Comment (marhcouto) [inline src/cloud_storage/README.md:11]: Maybe include this in dependencies install

**Review by marhcouto — CHANGES_REQUESTED** [review]
All seems good, just two more points:
- Include doxygen documentation strings in the functions, like the ones in Evaluator
- Sort the thing about the secret

**Review by marhcouto — APPROVED** [review]
Seems all good to me!

File: src/cloud_storage/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+ - **<csv_file_option>**: Specifies the source folder from which to read the CSV files.
+ - **<file1 file2 file3 | all>**: Specifies individual file names separated by spaces, or use all to upload all files in the folder.
+
+
+### Options for **<csv_file_option>**
+ - **state_est**: Reads time_exec measurements from unit tests of state estimation.
+ - **perception**: Reads time_exec measurements from unit tests of perception.
+ - **planning**: Reads time_exec measurements from unit tests of control.
```
Comment (Davide64-dev) [inline src/cloud_storage/README.md:23]: Small error here **planning** -> control. Check this

**Review by Davide64-dev — APPROVED** [review]
✅. Just a small note

**Pull request merged**